sdk/python/symbolchain/nem/TransactionFactory.py

```diff
@@ -86,7 +86,8 @@ def _build_rules(base_type_parsing_rules):
 		for name, typename in sdk_type_mapping.items():
 			factory.add_pod_parser(name, typename)
 
-		for name in ['struct:SizePrefixedMosaic', 'struct:SizePrefixedMosaicProperty', 'struct:SizePrefixedMultisigAccountModification']:
-			factory.add_array_parser(name)
+		arrays = ['SizePrefixedMosaic', 'SizePrefixedMosaicProperty', 'SizePrefixedMultisigAccountModification', 'SizePrefixedCosignature']
```

Comment:
consider renaming arrays => array_names

Reply:
ok, that's actually what I had earlier but it looked like it's crossing width boundary, but seems pycodestyle does not care 😁